app/Environment.php

```diff
@@ -28,91 +31,88 @@ class Environment
 
     public function __construct(DotenvFactory $dotenvFactory)
     {
-        $this->dotenvFactory = $dotenvFactory;
+        $this->envVariables = $dotenvFactory->create();
     }
 
-    public static function getKeys(): array
+    public function getKeys(): array
     {
         return static::$keys;
     }
 
-    public static function getValues(): array
+    public function getValues(): array
     {
         return array_only(getenv(), static::getKeys());
     }
 
-    public static function load(): void
+    public function load(): void
     {
-        static::loadEnv(static::getContextFwd());
+        static::loadEnv(static::getContextEnv('.fwd'));
         static::loadEnv(static::getContextEnv());
         static::loadEnv(static::getDefaultFwd());
 
         static::fixVariables();
     }
 
-    public static function getDefaultPath()
+    public function getDefaultPath()
     {
         return base_path();
     }
 
-    public static function getContextPath()
+    public function getContextPath()
     {
         return getcwd();
     }
 
-    public static function getDefaultDockerCompose()
+    public function getDefaultDockerCompose()
     {
         return static::getDefaultPath() . '/docker-compose.yml';
     }
 
-    public static function getContextDockerCompose()
+    public function getContextDockerCompose()
     {
         return static::getContextPath() . '/docker-compose.yml';
     }
 
-    public static function getDefaultFwd()
+    public function getDefaultFwd()
     {
         return static::getDefaultPath() . '/.fwd';
```

Comment:
below we are using sprintf for concat strings... maybe worth expanding the pattern to the whole class at least